templates/style.scss

```diff
@@ -141,14 +143,13 @@ div.container {
 }
 
 div.container-rustdoc {
-    max-width: 1200px;
```

Comment:
What does removing this rule do?

Reply:
After talking with Guillaume on Discord, it seems like this was the thing causing the bug Onur had seen, with the scrollbar appearing next to the content rather than on the edge of the viewpoint. However, removing this also causes the "crate search" box in the docs.rs navbar to move all the way to the side:

![image](https://user-images.githubusercontent.com/5217170/60114467-7f871500-9739-11e9-9eca-c27ef2501d43.png)

Maybe we can move this rule to `div.nav-container-rustdoc` instead, rather than deleting it entirely? The value itself may need to be changed, since the rustdoc content is a little wider than 1200px.